modules/apps/headless/headless-object/headless-object-impl/rest-openapi.yaml

```diff
@@ -1,5 +1,63 @@
 components:
     schemas:
+        Collaborator:
+            description:
+                Represents a collaborator for an entry.
+            properties:
+                actionIds:
+                    description:
+                        The collaborator actions for the shared asset.
+                    items:
+                        type: string
+                    type: array
+                actions:
+                    additionalProperties:
+                        additionalProperties:
```

Comment:
@Mikellorza, out of curiosity, why is there a nested additionalProperties here?

Reply:
if I'm honest I followed the same patter that we have for ObjectEntryFolder, for that reason I copied this field to the collaborator, but maybe if it is strange for you I can remove it. Let me know what do you think.

Reply:
Talking to @4lejandrito, better remove it

Reply:
not sure If I have to delete it, because I will need actions for following apis to manage each item, we do not  need to create them to use in our development but I talked with Adolfo to create them in order to manage each collaborator separately. So, I am going to delete actions for now, and then, I will create it if needed.

Reply:
done